config/environments/development.rb

```diff
@@ -93,4 +93,8 @@
 
   # Custom config
   config.bypass_filter_parameter_logging = ENV.fetch("ENABLE_BYPASS_FILTER_PARAMETER_LOGGING", false)
+
+  # Used to handle HTTP_X_WITH_SERVER_DATE header for simulating API
```

Comment:
Hi @ethax-ross Can't we add this only to `config/application.rb` instead of all envs?